pkg/virtual/workspaces/registry/rest.go

```diff
@@ -549,3 +570,124 @@ func (s *REST) Delete(ctx context.Context, name string, deleteValidation rest.Va
 
 	return nil, false, errorToReturn
 }
+
+type KubeconfigSubresourceREST struct {
```

Comment:
separate file maybe?

Reply:
Fixed in commit 2c97666d51d66ccc901f7d6f92992f5ba1412a5d